astropy/modeling/core.py

```diff
@@ -3384,14 +3490,28 @@ def make_subtree_dict(tree, nodepath, tdict, leaflist):
         tdict[nodepath] = (tree, leftmostind, rightmostind)
 
 
-_ORDER_OF_OPERATORS = [('|',), ('&',), ('+', '-'), ('*', '/'), ('**',)]
+_ORDER_OF_OPERATORS = [('fix_inputs',),('|',), ('&',), ('+', '-'), ('*', '/'), ('**',)]
 OPERATOR_PRECEDENCE = {}
 for idx, ops in enumerate(_ORDER_OF_OPERATORS):
     for op in ops:
         OPERATOR_PRECEDENCE[op] = idx
 del idx, op, ops
 
 
+def fix_inputs(modelinstance, values):
+    """
+    This function creates a compound model with one or more of the input values
+    of the input model assigned fixed values (scalar or array).
+    The values argument is expected to be a dictionary. If the key is an integer
+    it refers to the number of the input argument (0-based). If the key is a
+    string, it looks for an input of that name.
+    For example, if one defines g as Gaussian2D(1,2,3,4,5), then:
+    gv = fix_inputs(g, {0: 2.5}) results in a 1D function equivalent to
+    Gaussian2D(1, 2, 3, 4, 5)(x=2.5, y)
```

Comment:
Could you write this as a proper `Examples`  section.

Reply:
Also, since this is a user facing function add a `Parameters` section.

Reply:
done